Source/WebSocket/Android/AndroidWebSocketProvider.cpp

```diff
@@ -119,10 +123,14 @@ struct HttpClientWebSocket
         const jstring javaSubProtocol = env->NewStringUTF(subProtocol.c_str());
         if (HadException(env) || !javaSubProtocol)
         {
+            env->DeleteLocalRef(javaUri);
```

Comment:
Line 120, delete javauri?

Reply:
Also, here, delete javaSubProtocol?

Reply:
yes I resolved all comments in latest iteration, I overthink this in the first iteration